### PR TITLE
Replace argparser with click

### PR DESCRIPTION
**Pull request by vidok**

### CODE REVIEW

File: connectors/connectors_cli.py [hunk 1]
```diff
@@ -175,7 +178,13 @@
     },
 }
 
+# Main group
+def print_version(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(__version__)
 
+<<<<<<< Updated upstream
```
Comment (tutelaris) [inline connectors/connectors_cli.py:187]: ✂️

File: connectors/connectors_cli.py [hunk 2]
```diff
@@ -213,108 +222,114 @@ async def ensure_index_exists():
 
 
 async def create_connectors_index():
+=======
```
Comment (tutelaris) [inline connectors/connectors_cli.py:225]: ✂️

File: connectors/connectors_cli.py [hunk 3]
```diff
@@ -213,108 +222,114 @@ async def ensure_index_exists():
...
+    click.echo('test connector')
+
+cli.add_command(connector)
+
+
+# Index group
+@click.group(invoke_without_command=True)
+def index():
+    click.echo('testing index')
+>>>>>>> Stashed changes
```
Comment (tutelaris) [inline connectors/connectors_cli.py:254]: ✂️

File: connectors/connectors_cli.py [hunk 4]
```diff
@@ -213,108 +222,114 @@ async def ensure_index_exists():
...
+
+# Job group
+@click.group(invoke_without_command=True)
+def job():
+    click.echo('testing job')
+
+cli.add_command(job)
+
+
+# def run(args):
```
Comment (tutelaris) [inline connectors/connectors_cli.py:266]: Maybe it makes sense to remove these comments completely? It's really hard to read and review the code with it

Comment (vidok) [reply connectors/connectors_cli.py:225]: Doing it step by step. There are some parts I will re-use in the feature so I want to keep them in this file for now

**Pull request merged**